trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
Another strategy could be enforcing a rule of only one proto file per package. 
- We could collapse all the protos into a single file
- We could split the various proto files into separate packages.